extensions/geohash-encode-decode/package.json

```diff
@@ -0,0 +1,57 @@
+{
+  "$schema": "https://www.raycast.com/schemas/extension.json",
+  "name": "geohash-encode-decode",
+  "title": "Geohash",
+  "description": "Encode coordinates as geohash, or decode a geohash and get useful info",
+  "icon": "command-icon.png",
+  "author": "alrico88",
+  "license": "MIT",
+  "commands": [
+    {
+      "name": "encode-coordinates",
+      "title": "Encode as Geohash",
+      "description": "Enter a latitude and longitude and get that point's representation as a geohash for any precision",
+      "mode": "view"
+    },
+    {
+      "name": "decode-geohash",
+      "title": "Decode Geohash",
+      "description": "Enter a geohash and get insights such as the GeoJSON or WKT Polygon representation, centroid, neighbors, and more",
+      "mode": "view"
+    }
+  ],
+  "preferences": [
+    {
```

Comment:
I'm not really sure if we really need this preferences option. While doing the review I thought, why it didn't pop me to root once I performed an action, I started investigating and find in the code that there are some preferences that I didn't expect as a user. 
So it's not obvious that there are some preferences that change the behaviour after an action is performed.

I would actually always pop to root always once an action is performed, what is the case when you still need the list after you copied a value?
Or at least change the default value for the pref's option to `tru`

Reply:
Totally makes sense to at least have it as true, or, I can't see a use case where is makes sense to go back, maybe it's just because I don't regular use such thing in the first place 😄

Reply:
I had doubts regarding which value to default it to, and all in all I suppose it's better to make it auto close by default as you say. Changed it that way!